### PR TITLE
magento/devdocs#: Update escapers

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds `escapeJsQuote()`, `escapeQuote()`, `escapeXssInUrl()` into "Using the Escaper classes" section.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/extension-dev-guide/xss-protection.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/xss-protection.html

## Links to Magento source code

- [escapeJsQuote](https://github.com/magento/magento2/blob/2.2/lib/internal/Magento/Framework/Escaper.php#L293)
- [escapeQuote](https://github.com/magento/magento2/blob/2.2/lib/internal/Magento/Framework/Escaper.php#L349)
- [escapeXssInUrl](https://github.com/magento/magento2/blob/2.2/lib/internal/Magento/Framework/Escaper.php#L313)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/extension-dev-guide/xss-protection.md [hunk 1]
```diff
@@ -34,11 +34,16 @@ For more information, see the article on [templates XSS security]({{ page.baseur
...
-* `escapeCss()` - Used for escaping strings inside a {% glossarytooltip 6c5cb4e9-9197-46f2-ba79-6147d9bfe66d %}CSS{% endglossarytooltip %} context.
-* `escapeJs()` - Used for escaping strings inside a JavaScript context.
-* `escapeUrl()` - Used for escaping strings that will be used in a {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %}.
+Function | Description
+--- | ---
+`escapeHtml()` | Used for escaping string inside HTML content.
+`escapeHtmlAttr()` | Used for escaping strings in HTML tag attributes.
+`escapeCss()` | Used for escaping strings inside a {% glossarytooltip 6c5cb4e9-9197-46f2-ba79-6147d9bfe66d %}CSS{% endglossarytooltip %} context.
+`escapeJs()` | Used for escaping strings inside a JavaScript context.
+`escapeJsQuote()` | Used for escaping quotes in Java Script
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.2/extension-dev-guide/xss-protection.md]: **fixed**

File: guides/v2.2/extension-dev-guide/xss-protection.md [hunk 2]
```diff
@@ -34,11 +34,16 @@ For more information, see the article on [templates XSS security]({{ page.baseur
...
+`escapeQuote()` | Used for escaping quotes inside html attributes
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.2/extension-dev-guide/xss-protection.md]: **fixed**

@hguthrie , could you please review?

Thank you!

**Comment by hguthrie** [comment]
Some statements in the table end with a period and some don't. All statements in a list or table must follow the same convention.

**Comment by atwixfirster** [comment]
> Some statements in the table end with a period and some don't. All statements in a list or table must follow the same convention.

Good point, @hguthrie!

**fixed**

**Pull request merged**